src/jobs/NotificationTrigger/__tests__/NotificationTrigger.integration.js

```diff
@@ -1,6 +1,6 @@
 const NotificationTrigger = require('../index')
 
-jest.setTimeout(20000)
+jest.setTimeout(200000)
```

Comment:
@dev-siris Why are you increasing timeout here? Tests should be completed in few seconds. If it takes more than few seconds, then, it is the problem with the test.

Reply:
@syuraj  sir mero pc ma time badi lagi rako xa..

Reply:
@dev-siris it is something different issue, test shouldn't have even take 20 seconds